_data/toc/cloud-guide.yml

```diff
@@ -40,11 +39,9 @@ pages:
 
         - label: Bitbucket integration
           url: /cloud/project/bitbucket-integration.html
-          include_versions: ["2.0"]
 
         - label: GitHub integration
           url: /cloud/project/project-integrate-github.html
```

Comment:
So, if this is only included in 2.0 and we don't need it, then you have to remove all three lines. Try that!